Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -16,6 +16,23 @@ The Multiplayer Samples Co-op release provides Boss Room, a small-scale cooperat
 
 ### Known issues
 
-* After a round of Boss Room completes and a party member returns to the menu, it pulls all players and leader to menu.
-* The welcome player message in the lobby indicates P2 (player 2) regardless of your player number.
-* When multiple players select the same character and click **Ready** at the same time, it defaults to host and prevents the game from starting. This may be due to Unity MLAPI exceptions, requiring additional messaging to expose the information to the user and developer.
\ No newline at end of file
+The following issues may occur for access and games:
+
+* Sometimes when the host leaves the Boss Room, not all clients will return to the Main Menu, but will remain stuck in the Boss Room scene. 
+* Sometimes after completing a match and the host starts a new match from the Victory or Loss screen, connected players may have no visible interactions to join or select characters.
+* A player may encounter a rare exception when the Tank character uses her Shield Aura ability. This issue may be due to intercepting the Boss charge attack.
+* If two players in the Character Select **Ready** for the same hero at the same time, the UI will update to *Readied* on both clients, but only one will have actually selected the hero on the Host. This issue blocks Character Select from proceeding.
+* Any player that plays the game and then returns to the Main Menu may be unable to Start or Join properly again, requiring you to restart the client.
+* The Green Quads may show on impact when the Archer arrow strikes enemies. This issue may only occur in the editor.
+* You may encounter a game error due to a Unity MLAPI exception. Not all MLAPI exceptions are fully exposed few informing users.
+* The Photon Transport currently generates some errors in the Player log related to the `PhotonCryptoPlugin`.
+* The welcome player message in the lobby indicates P2 (player 2) regardless of your player number. Currently the Character Select scene displays “Player1” and “P1” in two locations, where it is intended that the user’s name be displayed.  
+* The spawner portal system requires updates to better handle wave spawning logic and destroying crystal game elements.
+* Players may not reliably play another match when selecting **Return to Main Menu** during the post-game scene. THis may be due to states not properly clearing.
```

Comment:
```suggestion
* Players may not reliably play another match when selecting **Return to Main Menu** during the post-game scene. This may be due to states not properly clearing.
```